### PR TITLE
Update other-hadoop.md

### PR DESCRIPTION
**Pull request by mspangdal**
fixes a broken link

### CODE REVIEW

File: docs/content/operations/other-hadoop.md [hunk 1]
```diff
@@ -101,7 +101,7 @@ Another workaround solution is to build a custom fat jar of Druid using [sbt](ht
 
 (2) Make a new directory named 'druid_build'.
 
-(3) Cd to 'druid_build' and create the build.sbt file with the content [here](./use_sbt_to_build_fat_jar.html).
+(3) Cd to 'druid_build' and create the build.sbt file with the content [here](./use_sbt_to_build_fat_jar.md).
```
Comment (nishantmonu51) [inline docs/content/operations/other-hadoop.md]: I think this should be .html instead of .md

Comment (mspangdal) [inline docs/content/operations/other-hadoop.md]: https://github.com/druid-io/druid/blob/master/docs/content/operations/use_sbt_to_build_fat_jar.html gives me a 404

**Comment by fjy** [comment]
@mspangdal the relative links are for the website

http://druid.io/docs/latest/operations/use_sbt_to_build_fat_jar.html

**Comment by mspangdal** [comment]
Okay

**Pull request closed without merging**